uniffi_bindgen/src/bindings/python/templates/ObjectTemplate.py

```diff
@@ -60,9 +63,32 @@ def __ne__(self, other: object) -> {{ ne.return_type().unwrap()|type_name }}:
         return {{ ne.return_type().unwrap()|lift_fn }}({% call py::to_ffi_call_with_prefix("self._pointer", ne) %})
 {%-         when UniffiTrait::Hash { hash } %}
             {%- call py::method_decl("__hash__", hash) %}
-{%      endmatch %}
-{% endfor %}
+{%-      endmatch %}
+{%- endfor %}
+
+{%- if ci.is_name_used_as_error(name) %}
+{# Due to some mismatches in the ffi converter mechanisms, errors are forced to be a RustBuffer #}
+class {{ ffi_converter_name }}__as_error(_UniffiConverterRustBuffer):
```

Comment:
How about defining something like Kotlin's [lowerIntoRustBuffer()](https://github.com/mozilla/uniffi-rs/blob/d15d530439b65e9d8f9c6e0f8847306388ac7a4a/uniffi_bindgen/src/bindings/kotlin/templates/FfiConverterTemplate.kt#L34-L47) rather than a second FFIConverter class.  As that method mentions, it's also what we need to do for [callback interfaces](https://github.com/mozilla/uniffi-rs/blob/d15d530439b65e9d8f9c6e0f8847306388ac7a4a/uniffi_bindgen/src/bindings/python/templates/CallbackInterfaceTemplate.py#L37-L39), so having that function could simplify that code a bit to.